terraform/deployments/search-api-v2/modules/google_discovery_engine_restapi/main.tf

```diff
@@ -67,19 +67,52 @@ resource "restapi_object" "discovery_engine_serving_config" {
   })
 }
 
-# Future serving config managed by Search Admin
-resource "restapi_object" "discovery_engine_serving_config_site_search" {
```

Comment:
Note that this will cause the `site_search` serving config to be removed.

We don't actually use this anywhere (the real serving config in live use is `default_search` above), so this is fine.